src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/BulkTokenStringCompositeSearchParameterV1RowGenerator.cs

```diff
@@ -36,7 +37,7 @@ internal class BulkTokenStringCompositeSearchParameterV1RowGenerator : BulkCompo
                 row = new BulkTokenStringCompositeSearchParamTableTypeV1Row(
                     offset,
                     searchParamId,
-                    token1Row.SystemId,
+                    token1Row.SystemId ?? SqlSearchConstants.NullId,
```

Comment:
nit: Do we need to update here since as part of this PR we will be using BulkTokenStringCompositeSearchParamTableTypeV2Row and not V1?

Reply:
Other RowGenerators will use Id as 0 going forward with this change even if the version is < 26 so to be consistent with that applied changes here as well

Reply:
that would create case where you have half of database filled with nulls and half of database filled with 0, and i'm not what search condition we will use, but it would be one and not composite of them I presume.